lib/bolt/cli.rb

```diff
@@ -413,7 +417,14 @@ def list_plans
 
     def list_targets
       update_targets(options)
-      outputter.print_targets(options)
+      targets = options[:targets].map(&:name)
```

Comment:
Why did this get moved here? It seems like if we ever want to call `print_targets` elsewhere we'll want it to have the same format.

Reply:
Updated.